metabase/metabase
# Migrate frontend/src/metabase/components/Popover/Popover.module.css

Epic #38811
task https://github.com/metabase/metabase/issues/40722

### Description

Migrate `frontend/src/metabase/components/Popover/Popover.module.css`

### How to verify

Open dashboard widget to see if it's still looking fine
![image](https://github.com/metabase/metabase/assets/1937582/88834308-5e98-4d92-a3a9-a56f2054d9b0)
